examples/convert.rs

```diff
@@ -115,8 +123,14 @@ fn main() {
         #[cfg(feature = "glsl")]
         "comp" => {
             let input = fs::read_to_string(&args[1]).unwrap();
-            naga::front::glsl::parse_str(&input, "main".to_string(), naga::ShaderStage::Compute)
-                .unwrap()
+            naga::front::glsl::parse_str(
+                &input,
+                "main".to_string(),
+                naga::ShaderStage::Compute {
+                    local_size: (8, 1, 1),
```

Comment:
I think GLSL should know which group size it operates with?

Reply:
Certainly not at the point of calling that function. That API needs some fixing now.

Reply:
If it doesn't matter, let's put some value indicating this, i.e. (0,0,0).
maybe make the `local_size` to be a fixed-size array even? so that we can put `[0; 3]`